### PR TITLE
community/php7: fix license, use https, fix include_path setting

### PR DESCRIPTION
**Pull request by andypost**
- https://secure.php.net/license/index.php
- setting is broken, thanks to @jirutka

### CODE REVIEW

File: community/php7/APKBUILD [hunk 1]
```diff
@@ -26,15 +26,15 @@
...
 _apiver=20170718
 _suffix=${pkgname#php}
 # Is this package the default (latest) PHP version?
 _default_php="yes"
 pkgdesc="The PHP$_suffix language runtime engine"
-url="http://www.php.net/"
+url="https://secure.php.net/"
 arch="all"
-license="PHP-3.0 BSD LGPL-2.0 MIT Zend"
+license="PHP-3.01"
```
[the remark this answers is not in the document]
Comment (andypost) [reply community/php7/APKBUILD]: Yep, you are right there's more then that list - checked in http://metadata.ftp-master.debian.org/changelogs/main/p/php7.2/php7.2_7.2.4-1_copyright

**Comment by jirutka** [comment]
I've merged it without the license change. Thanks!

**Comment by jirutka** [comment]
I've also backported it to 3.8-stable.

File: community/php7/APKBUILD (hunk 1 above)
[the remark this answers is not in the document]
Comment (andypost) [reply community/php7/APKBUILD]: Checked all libs & updated PR

**Pull request closed without merging**